adal/src/androidTest/java/com/microsoft/aad/adal/PackageHelperTests.java

```diff
@@ -44,23 +50,27 @@
 import javax.crypto.SecretKeyFactory;
 import javax.crypto.spec.PBEKeySpec;
 import javax.crypto.spec.SecretKeySpec;
+import static org.junit.Assert.*;
 
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-public class PackageHelperTests extends AndroidTestCase {
+@RunWith(AndroidJUnit4.class)
+public class PackageHelperTests {
 
     private static final int TEST_UID = 13;
 
     private byte[] mTestSignature;
 
     private String mTestTag;
 
+    private Context mContext;
+
     @SuppressLint("PackageManagerGetSignatures")
-    protected void setUp() throws Exception {
-        super.setUp();
-        getContext().getCacheDir();
-        System.setProperty("dexmaker.dexcache", getContext().getCacheDir().getPath());
+    @Before
+    public void setUp() throws Exception {
+        mContext = InstrumentationRegistry.getContext();
```

Comment:
same comment about being consistent on how you handle InstrumentationRegistry.getContext